config/features.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+Flipflop.configure do
+  # Strategies will be used in the order listed here.
+  strategy :cookie
+  strategy :active_record
+  strategy :default
+
+  group :lewis_library do
+    feature :lewis_staff, title: 'Staff features', description: 'Allow Lewis staff members to administer Lewis lockers and applications'
```

Comment:
Should we add default values for these features? I'd be tempted to default to `false` right now

```suggestion
    feature :lewis_staff, title: 'Staff features', description: 'Allow Lewis staff members to administer Lewis lockers and applications', default: false
```